### PR TITLE
debt: review implicit public fields that are actually protected (#166603)

### PR DESCRIPTION
**Pull request by bpasero**

### CODE REVIEW

File: src/vs/workbench/contrib/terminal/browser/terminalView.ts [hunk 1]
```diff
@@ -129,7 +129,8 @@ export class TerminalViewPane extends ViewPane {
 		}
 	}
 
-	override renderBody(container: HTMLElement): void {
+	// eslint-disable-next-line @typescript-eslint/naming-convention
```
Comment (sandy081) [inline src/vs/workbench/contrib/terminal/browser/terminalView.ts:132]: Wondering why only this line has to be disabled and not others

**Review by sandy081 — APPROVED** [review]
LGTM except for the comment I added.

File: src/vs/workbench/contrib/terminal/browser/terminalView.ts (hunk 1 above)
Comment (bpasero) [reply src/vs/workbench/contrib/terminal/browser/terminalView.ts:132]: Yeah good point, was wondering the same @Tyriar

Comment (Tyriar) [reply src/vs/workbench/contrib/terminal/browser/terminalView.ts:132]: Because terminal has an eslintrc file scopes to the folder to enforce some rules

**Pull request merged**